llvm/test/tools/llc/new-pm/machine-branch-prob.ll

```diff
@@ -0,0 +1,35 @@
+; RUN: llc -mtriple=x86_64-linux-gnu %s -stop-after=finalize-isel -o - | \
```

Comment:
Apparently llc/new-pm/ has a requires X86 which is very surprising to me. In a follow up commit, should really move these under an X86 directory or use individual REQUIREs in the tests

Reply:
+1